src/Acuminator/Acuminator.Analyzers/StaticAnalysis/Descriptors.cs

```diff
@@ -412,5 +412,11 @@ public static class Descriptors
 
 		public static DiagnosticDescriptor PX1095_PXDBScalarMustBeAccompaniedNonDBTypeAttribute { get; } =
 			Rule("PX1095", nameof(Resources.PX1095Title_PXDBScalar).GetLocalized(), Category.Default, DiagnosticSeverity.Error, DiagnosticsShortName.PX1095PXDBScalar);
+
```

Comment:
The severity of both diagnostic should be Error

Reply:
Done, thank you.